### ISSUE TITLE
installVueDevtools() hangs up in authentication proxy environment

### ISSUE THREAD

**Issue by vmi** [nklayman/vue-cli-plugin-electron-builder#379]
**Describe the bug**
installVueDevtools() hangs up in authentication proxy environment.

**To Reproduce**
In authentication proxy environment, I ran the following command:
```
vue create test
cd test
vue add electron-builder
yarn electron:serve
```

**Expected behavior**
I expected the screen to be displayed, but nothing was displayed and it hung up.

**Environment (please complete the following information):**
 - OS and version: Windows 10 Ent 1709 (64bit) 
 - node version: v12.7.0
 - npm version: 6.10.0
 - yarn version (if used): 1.17.3
 - vue-cli-plugin-electron-builder version : 1.3.6
 - electron version: 5.0.8
 - other vue plugins used: 
 - custom config for vcp-electron-builder: 
 - (if possible) link to your repo: 

**Additional context**
When the following patch was applied, it worked correctly:
```
> diff -u node_modules/vue-cli-plugin-electron-builder/lib/installVueDevtools/utils.js.*
--- node_modules/vue-cli-plugin-electron-builder/lib/installVueDevtools/utils.js.ORIG   2019-07-26 18:36:52.982350000 +0900
+++ node_modules/vue-cli-plugin-electron-builder/lib/installVueDevtools/utils.js.PATCHED   2019-07-26 18:39:39.288531300 +0900
@@ -15,6 +15,9 @@
 export const downloadFile = (from, to) =>
   new Promise((resolve, reject) => {
     const req = request(from)
+    req.on('login', (authInfo, callback) => {
+      callback('PROXY_USER', 'PROXY_PASSWORD')
+    })
     req.on('response', res => {
       // Shouldn't handle redirect with `electron.net`, this is for https.get fallback
       if (
```

**Comment by ChrisKader** [comment]
It seems that the built in [request ](https://electronjs.org/docs/api/net) client for electron does not use the built in NodeJS library (or process vars) and you have to actually specify proxy credentials, as you did in your patch.

https://electronjs.org/docs/api/client-request

It would probably be good for you to submit a PR with your patch so it can be merged in.

**Comment by nklayman** [comment]
The fix will be available in the next v2.0 beta, just set `process.env.VUE_DEVTOOLS_PROXY_USER` and `process.env.VUE_DEVTOOLS_PROXY_PASSWORD`.
